app/controllers/api/v1/rooms_controller.rb

```diff
@@ -44,11 +44,24 @@ def start
         end
       end
 
+      # POST /api/v1/rooms.json
+      # Returns: { data: Array[serializable objects] , errors: Array[String] }
+      # Does: creates a room for the authenticated user.
+      def create
+        room = Room.new(room_params.merge(user_id: current_user.id)) and room.save!
```

Comment:
This syntax is weird. Remember that keeping things simple makes life much easier for us

```
Room.create!(room_params.merge(user_id: current_user.id))
```

Reply:
You got reason, 
It's funny though that I used it at first but don't why I've changed it.